legend-pure-m2-store-relational/src/main/resources/platform/relational/relational.pure

```diff
@@ -390,97 +390,114 @@ Class meta::relational::metamodel::RelationalOperationElementWithJoin extends me
 {
     size: Integer[1];
 }
 
-Class meta::relational::metamodel::datatype::Varbinary extends meta::relational::metamodel::datatype::DataType
+Class meta::relational::metamodel::datatype::Varbinary extends meta::relational::metamodel::datatype::CoreDataType
 {
     size: Integer[1];
 }
 
-Class meta::relational::metamodel::datatype::Decimal extends meta::relational::metamodel::datatype::DataType
+Class meta::relational::metamodel::datatype::Decimal extends meta::relational::metamodel::datatype::CoreDataType
 {
     precision : Integer[1];
     scale     : Integer[1];
 }
 
-Class meta::relational::metamodel::datatype::Numeric extends meta::relational::metamodel::datatype::DataType
+Class meta::relational::metamodel::datatype::Numeric extends meta::relational::metamodel::datatype::CoreDataType
 {
     precision : Integer[1];
     scale     : Integer[1];
 }
 
-Class meta::relational::metamodel::datatype::Timestamp extends meta::relational::metamodel::datatype::DataType
+Class meta::relational::metamodel::datatype::Timestamp extends meta::relational::metamodel::datatype::CoreDataType
 {
 }
 
-Class meta::relational::metamodel::datatype::Date extends meta::relational::metamodel::datatype::DataType
+Class meta::relational::metamodel::datatype::Date extends meta::relational::metamodel::datatype::CoreDataType
 {
 }
 
-Class meta::relational::metamodel::datatype::Distinct extends meta::relational::metamodel::datatype::DataType
+Class meta::relational::metamodel::datatype::Distinct extends meta::relational::metamodel::datatype::CoreDataType
 {
 }
 
-Class meta::relational::metamodel::datatype::Other extends meta::relational::metamodel::datatype::DataType
+Class meta::relational::metamodel::datatype::Other extends meta::relational::metamodel::datatype::CoreDataType
 {
 }
 
-Class meta::relational::metamodel::datatype::Bit extends meta::relational::metamodel::datatype::DataType
+Class meta::relational::metamodel::datatype::Bit extends meta::relational::metamodel::datatype::CoreDataType
 {
 }
 
-Class meta::relational::metamodel::datatype::Binary extends meta::relational::metamodel::datatype::DataType
+Class meta::relational::metamodel::datatype::Binary extends meta::relational::metamodel::datatype::CoreDataType
 {
    size:Integer[1];
 }
 
-Class meta::relational::metamodel::datatype::Real extends meta::relational::metamodel::datatype::DataType
+Class meta::relational::metamodel::datatype::Real extends meta::relational::metamodel::datatype::CoreDataType
 {
 }
 
-Class meta::relational::metamodel::datatype::Array extends meta::relational::metamodel::datatype::DataType
+Class meta::relational::metamodel::datatype::Array extends meta::relational::metamodel::datatype::CoreDataType
 {
 }
 
+// Db Specific DataTypes
+// Big Query DataTypes
+Class meta::relational::metamodel::datatype::BigQueryString extends meta::relational::metamodel::datatype::DbSpecificDataType
```

Comment:
What about the numeric BigQuery types ? https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#numeric_types

Reply:
we were thinking of adding other types once the basic framework is in